docs/basics.rst

```diff
@@ -136,7 +136,13 @@ On Linux, the above example can be compiled using the following command:
 
 .. code-block:: bash
 
-    $ c++ -O3 -Wall -shared -std=c++11 -fPIC `python3 -m pybind11 --includes` example.cpp -o example`python3-config --extension-suffix`
+    $ c++ -O3 -Wall -shared -std=c++11 -fPIC $(python3 -m pybind11 --includes) example.cpp -o example$(python3-config --extension-suffix)
+
+.. note::
+
+    If you used :ref:`include_as_a_submodule` to install the pybind11 source,
+    then use ``$(python3-config --includes) -Iextern/pybind11/include``
```

Comment:
`-I` should really be `-isystem`.

Reply:
I agree, but the pybind11-config and the Python-config both use `-I` I believe, so it would be a little odd if this causes pybind11 to be system but not Python.h.

Reply:
```suggestion
    then use ``$(python3-config --includes) -isystem extern/pybind11/include``
```
For a simple example, not sure this is best, but it's also better to show the correct thing if it's visible.

Reply:
Hm... You're right that `pybind11-config` and `python-config` both use `-I`. I'd argue that both of those are wrong. I personally use `-isystem /usr/include/python3.9` and my cmake adds both with `SYSTEM`.

Anyway, I'm not super determined to change this `-I` to `-isystem`.